.gitignore

```
@@ -5,6 +5,10 @@
 # rather than matching a pattern. This helps makes it easier for us to know when
 # it's okay to remove lines from this file.
 
+# Secrets
+auth_config.json
+auth_config*.json
```

Can you use this instead? See comment above.

```suggestion
/auth_config.json
/auth_config*.json
```

Let me know if that won't work in this case for some reason!